spring-cloud-gcp-examples/spring-cloud-gcp-sql-example/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.cloud.gcp.sql.database-name=example
```

Comment:
example -> [database-name], or just instruct the users to create a database called "example"